python/django-ninja/requirements.txt

```diff
@@ -0,0 +1,4 @@
+django>=3.1.0,<3.2.0
+django-ninja==0.3.1
```

Comment:
could you use a more relax constraint ? e.g ->  `django-ninja>=0.3.1,<0.4.0`

Reply:
done